lib/api/2.0/notification.js

```diff
@@ -30,7 +36,124 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishs it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
+    /* The alert would look like the following:
```

Comment:
Do we want this big block of commented code to show an example, or was this just for testing purposes?